lib/lib_rss.php

```diff
@@ -81,6 +81,14 @@ function safe_ascii($text) {
 	return filter_var($text, FILTER_DEFAULT, FILTER_FLAG_STRIP_LOW | FILTER_FLAG_STRIP_HIGH);
 }
 
+if (function_exists('mb_convert_encoding')) {
+	function safe_utf8($text) { return mb_convert_encoding($text, 'UTF-8', 'UTF-8'); }
+} elseif (function_exists('iconv')) {
+	function safe_utf8($text) { return iconv('UTF-8', 'UTF-8//IGNORE', $text); }
+} else {
+	function safe_utf8($text) { return $text; }
```

Comment:
Should it also write a little warning in the logs about the fact that it's not actually doing anything at all?

Reply:
Maybe in the *check installation* page. Otherwise this would give one line in the logs per call

Reply:
True, that'd become excessive quickly.

Reply:
We already have a check there https://github.com/FreshRSS/FreshRSS/blob/91cb165829badde07c42a002215cf52779d891b6/lib/lib_install.php#L44

Reply:
They don't say what kind of problem you might get from not having it though, just a fairly vague "for Unicode". Assuming it shouldn't simply be required if you get such curious hard to track down issues from it?

Reply:
Indeed, we could improve the message and/or documentation